Extension/src/githubAPI.ts

```diff
@@ -35,7 +35,7 @@ export interface Build {
 * @param vsixName The name of the VSIX to search for
 * @return The download URL of the VSIX
 */
-function getVsixDownloadUrl(build: Build, vsixName: string): string {
+export function getVsixDownloadUrl(build: Build, vsixName: string): string {
```

Comment:
It didn't look like this was used anywhere else, so I don't think it needs to be `export`